docs/tutorial/install.rst

```diff
@@ -0,0 +1,40 @@
+Install Microceph using LXD
+======================================================
+
+
+Setup VMs
+~~~~~~~~~
+
+.. code-block:: shell
+
+    lxc launch images:ubuntu/22.04/cloud microceph-1 --vm
+    lxc launch images:ubuntu/22.04/cloud microceph-2 --vm
+    lxc launch images:ubuntu/22.04/cloud microceph-3 --vm
+
+Add storage
+~~~~~~~~~~~
+
+.. code-block:: shell
+
+    for i in $(seq 1 3); do
+        lxc storage volume create default osd-$i --type block size=10GiB
+        lxc config device add microceph-$i osd-$i disk pool=default source=osd-$i
```

Comment:
Does this need any previous configuration setting previously set for it to work? I seem to recall that it did, but I may be wrong.